galaxy/api/views/views.py

```diff
@@ -51,6 +51,13 @@
 from galaxy.main import models
 from galaxy.common import version, sanitize_content_name
 
+# monitor
+from kombu import Connection as RabbitMQ
```

Comment:
Please organize imports according to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports):

> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

Reply:
@cutwater I am not familiar, is this about the order it's written?

Reply:
Yes, it's about the order.